modGui/src/org/aion/gui/controller/HeaderPaneControls.java

```diff
@@ -71,10 +72,19 @@ protected void registerEventBusConsumer() {
     }
 
     public void openAionWebSite() {
+        // borrowed from aion_ui's URLManager; when wallet version of modGui
+        // is merged, this will be replaced with a call to URLManager instead.
         try {
-            Desktop.getDesktop().browse(new URI(AION_URL));
-        } catch (IOException | URISyntaxException e) {
-            log.error("Exception occurred trying to open website: %s", e.getMessage(), e);
+            URI uri = new URI(AION_URL);
+            final String os = System.getProperty("os.name").toLowerCase();
+            if (os.contains("win")) {
+                Desktop.getDesktop().browse(uri);
+            } else if (os.contains("nix") || os.contains("nux") || os.indexOf("aix") > 0)
+                if (Runtime.getRuntime().exec(new String[]{"which", "xdg-open"}).getInputStream().read() != -1) {
+                    Runtime.getRuntime().exec(new String[]{"xdg-open", AION_URL});
```

Comment:
From its JavaDocs, it can also throw SecurityException -- we should probably catch that.  It cna also throw NullPointerException or IllegalArgumentException but I don't think we need to catch those (only happens if you pass it null or empty as the command)